### PR TITLE
Text responses

### PR DESCRIPTION
**Pull request by g4s8**
#210 - plain text responses with encoding from `CharSequence`

### CODE REVIEW

File: src/main/java/com/artipie/http/rs/common/RsText.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    public RsText(final CharSequence text, final Charset encoding) {
+        this.text = text;
+        this.encoding = encoding;
+    }
+
+    @Override
+    public CompletionStage<Void> send(final Connection connection) {
+        final byte[] bytes = this.text.toString().getBytes(this.encoding);
+        return connection.accept(
+            RsStatus.OK,
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/http/rs/common/RsText.java:76]: @g4s8 I'd extract field from response status and create ctor with default `200 OK` status

File: src/test/java/com/artipie/http/rs/common/RsTextTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        );
+    }
+
+    @Test
+    void headersHasContentSize() {
+        MatcherAssert.assertThat(
+            new StatefulResponse(new RsText("four")),
+            new RsHasHeaders(
+                Matchers.anything(),
+                Matchers.equalTo(new Header("Content-Length", "4"))
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/http/rs/common/RsTextTest.java:57]: @g4s8 could you please switch matchers to make meaningful matcher come first?

**Comment by g4s8** [comment]
@olenagerasimova thanks, please check new changes (also, I've refactored `RsJson` to accept additional `Response` parameter

**Comment by olenagerasimova** [comment]
@g4s8 thanks, looks good to merge

**Comment by 0crat** [comment]
Job `gh:artipie/http#213` is not assigned, can't get performer

**Pull request merged**